### PR TITLE
StRefMultCorr updated for standalone compilation

### PR DESCRIPTION
**Pull request by nigmatkulov**
Add README.md (some basic info) and LinkDef file (needed for standalone compilation)

### CODE REVIEW

**Review by marrbnl — APPROVED** [review]

**Comment by zsweger** [comment]
I think line 9 of StRefMultCorrLinkDef.h should be changed from 
#pragma link C++ class StCentralityMaker+
to 
#pragma link C++ class CentralityMaker+

I cloned this branch and attempted to do a standalone make, but it only worked after this change.

File: StRoot/StRefMultCorr/StRefMultCorrLinkDef.h [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#ifdef __CINT__
+
+#pragma link off all globals;
+#pragma link off all classes;
+#pragma link off all functions;
+
+// Main PicoDst classes
```
Comment (veprbl) [inline StRoot/StRefMultCorr/StRefMultCorrLinkDef.h]: Misleading comment
```suggestion
```

File: StRoot/StRefMultCorr/StRefMultCorrLinkDef.h [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+#pragma link C++ class StRefMultCorr+;
+#pragma link C++ class StCentralityMaker+;
```
Comment (veprbl) [inline StRoot/StRefMultCorr/StRefMultCorrLinkDef.h]: As defined in https://github.com/star-bnl/star-sw/blob/101589368fd7e154f2c263cbe282e276470b3b4c/StRoot/StRefMultCorr/CentralityMaker.h#L91
```suggestion
#pragma link C++ class CentralityMaker+;
```

**Review by zsweger — APPROVED** [review]

File: StRoot/StRefMultCorr/StRefMultCorrLinkDef.h [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+#endif
```
Comment (plexoos) [inline StRoot/StRefMultCorr/StRefMultCorrLinkDef.h:11]:
```suggestion

#endif
```

**Review by plexoos — APPROVED** [review]

**Pull request merged**